src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -17,14 +17,12 @@ language_settings:
 interfaces:
 - name: google.example.library.v1.LibraryService
   collections:
-  - name_pattern: archives/{archive_path=**}/books/{book}
-    entity_name: book
   - name_pattern: archives/{archive}/books/{book_path=**}
     entity_name: book_path
   - name_pattern: bookShelves/{book_shelf}
     entity_name: book_shelf
   - name_pattern: bookShelves/{book_shelf}/books/{book}
-    entity_name: book_2
+    entity_name: book
```

Comment:
We're now losing our testing of multiple resources having the same last path component